### PR TITLE
fix hero section

### PR DESCRIPTION
**Pull request by Ruslan-Dolibskyi**

### CODE REVIEW

File: src/css/hero.css [hunk 1]
```diff
@@ -76,14 +81,17 @@
 
 .hero-hashtags {
   display: flex;
+  flex-wrap: wrap;
```
Comment (LadaM) [inline src/css/hero.css:84]: Has to be this way:
image
Is this way:
image

Seems like hashtags should be aligned with the text. See the picture.

**Review by LadaM — CHANGES_REQUESTED** [review]

File: src/css/hero.css [hunk 2]
```diff
@@ -76,14 +81,17 @@
...
-  padding: 4px 12px 4px 12px;
+  padding: 4px 12px;
   border-radius: 30px;
   border: 1px solid var(--dark);
   font-size: 14px;
+  line-height: 1.28571;
+  letter-spacing: -0.02em;
 }
 
 .hero-widget {
```
Comment (LadaM) [inline src/css/hero.css:97]: Do you have an empty style?

File: src/css/hero.css [hunk 3]
```diff
@@ -76,14 +81,17 @@
```
Comment (LadaM) [inline src/css/hero.css:81]: I don't know where the class that makes space between the image and the hashtags is, but for tablet the bottom margin is wrong:
image

File: src/css/hero.css [hunk 4]
```diff
@@ -76,14 +81,17 @@
 
 .hero-hashtags {
   display: flex;
+  flex-wrap: wrap;
   gap: 8px;
 }
 
 .hero-hashtags-item {
-  padding: 4px 12px 4px 12px;
+  padding: 4px 12px;
```
Comment (LadaM) [inline src/css/hero.css:89]: Copied from Figma:
width: Hug (86px)px;
height: Hug (32px)px;
**padding:** 4px **16px** 4px **16px**;
gap: 10px;
border-radius: 30px 0px 0px 0px;
border: 1px 0px 0px 0px;
opacity: 0px;

File: src/css/hero.css (hunk 1 above)
Comment (Ruslan-Dolibskyi) [reply src/css/hero.css:84]: done

**Review by LadaM — APPROVED** [review]
Hoping that you've checked everything ;)

**Pull request merged**